packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -31,6 +31,7 @@ const Map<String, String> _kManuallyPinnedDependencies = const <String, String>{
   // TODO(amirh): unpin this.
   'test': '0.12.26',
   // Add pinned packages here.
+  'url_launcher': '1.0.3',
```

Comment:
you should give a comment, with a TODO, as with the 'test' package above or as in the example in the documentation above.

Reply:
(ideally the comment would link to the bug report explaining why it's pinned)